dbms/src/Common/ColumnsHashingImpl.h

```diff
@@ -36,10 +36,12 @@ struct LastElementCache
     bool empty = true;
     bool found = false;
 
+    IGNORE_MAYBE_UNINITIALIZED_PUSH
```

Comment:
We can probably remove this one as well, I tried and there's no warning.

Reply:
Removing that leads to
```
/home/amos/git/chorigin/dbms/src/Common/ColumnsHashingImpl.h:40:54: error: 'state' may be used uninitialized in this function [-Werror=maybe-uninitialized]
     bool check(const Value & value_) { return !empty && value == value_; }
                                               ~~~~~~~^~~~~~~~~~~~~~~~~~
```